preamble.tex

```diff
@@ -1,21 +1,15 @@
 \usepackage[margin=1in]{geometry}
-\usepackage{amsmath, amssymb}
-\usepackage{amstext} % for \text macro
-\usepackage{array}   % for new column macro
```

Comment:
Why not these packages?

Reply:
Mathtools is a package combining & extending most ams packages. Importing the first 3 would therefore be redundant. If we import mathtool, we simply need amssymb here. 
as for array, it's already emulated by memoir